libs/api/domains/health-insurance-v2/src/lib/graphql/models/accidentNotificationAttachment.model.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { IsEnum } from 'class-validator'
+
+export enum AttachmentTypes {
+  INJURY_CERTIFICATE = 'InjuryCertificate',
+  PROXY_DOCUMENT = 'ProxyDocument',
+  POLICE_REPORT = 'PoliceReport',
+}
+
+@ObjectType()
+export class AccidentNotificationAttachment {
```

Comment:
Should follow the [graphql naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api).